### PR TITLE
[sub]fix(backend/image_transfert/encoder): update pydantic method

### PR DESCRIPTION
**Pull request by thbcmlowk**
## Description

Closes FL-1242

## Checklist

- [ ] [changelog](../CHANGELOG.md) was updated with notable changes
- [ ] documentation was updated

### CODE REVIEW

File: backend/image_transfer/encoder.py [hunk 1]
```diff
@@ -133,7 +133,7 @@ def create_zip_from_docker_images(
         dest = payload_descriptor.manifests_paths[manifest.docker_image_name]
         zip_file.writestr(dest, manifest.content)
 
-    zip_file.writestr("payload_descriptor.json", payload_descriptor.json(indent=4))
+    zip_file.writestr("payload_descriptor.json", payload_descriptor.model_dump_json(indent=4))
```
Comment (ThibaultFy) [inline backend/image_transfer/encoder.py:136]: You need to change also `PayloadDescriptor.parse_raw` into `PayloadDescriptor.model_validate_json` in `decoder.py`

Comment (thbcmlowk) [reply backend/image_transfer/encoder.py:136]: Nice catch! You mean [here](https://github.com/Substra/substra-backend/blob/fix/pydantic-updates/backend/image_transfer/decoder.py#L165) and like this, right?
```diff
-     return PayloadDescriptor. parse_raw(zip_file.read("payload_descriptor.json").decode())
+    return PayloadDescriptor.model_validate_json(zip_file.read("payload_descriptor.json").decode())
```

Comment (ThibaultFy) [reply backend/image_transfer/encoder.py:136]: yep ! (I did it for the PR refactor image transfer that's why ^^")

Comment (thbcmlowk) [reply backend/image_transfer/encoder.py:136]: Also made the pydantic upgrade, but still 🙂

Comment (thbcmlowk) [reply backend/image_transfer/encoder.py:136]: Done!

**Review by ThibaultFy — APPROVED** [review]
perfect :)

**Pull request merged**